ks/azure/centos80.ks

```diff
@@ -0,0 +1,193 @@
+# Kickstart for provisioning a CentOS 8.0 Azure VM
+
+# System authorization information
+auth --enableshadow --passalgo=sha512
+
+# Use graphical install
+text
+
+# Do not run the Setup Agent on first boot
+firstboot --disable
+
+# Keyboard layouts
+keyboard --vckeymap=us --xlayouts='us'
+
+# System language
+lang en_US.UTF-8
+
+# Network information
+network --bootproto=dhcp
+
+# Use network installation
+url --url="http://olcentgbl.trafficmanager.net/centos/8.0.1905/BaseOS/x86_64/os/"
+repo --name "BaseOS" --baseurl="http://olcentgbl.trafficmanager.net/centos/8.0.1905/BaseOS/x86_64/os/" --cost=100
+repo --name="AppStream" --baseurl="http://olcentgbl.trafficmanager.net/centos/8.0.1905/AppStream/x86_64/os/" --cost=100
+
+# Root password
+rootpw --plaintext "to_be_disabled"
+
+# System services
+services --enabled="sshd,waagent,NetworkManager,systemd-resolved"
+
+# System timezone
+timezone Etc/UTC --isUtc
+
+# Partition clearing information
+clearpart --all --initlabel
+
+# Clear the MBR
+zerombr
+
+# Disk partitioning information
+part /boot --fstype="xfs" --size=500
+part / --fstype="xfs" --size=1 --grow --asprimary
+
+# System bootloader configuration
+bootloader --location=mbr --timeout=1
+
+# Firewall configuration
+firewall --disabled
+
+# Enable SELinux
+selinux --enforcing
+
+# Don't configure X
+skipx
+
+# Power down the machine after install
+poweroff
+
+# Disable kdump
+%addon com_redhat_kdump --disable
+%end
+
+%packages
+WALinuxAgent
+@base
+@core
+#@container-tools
+chrony
+sudo
+parted
+-dracut-config-rescue
+-postfix
+-NetworkManager-config-server
+openssh-server
+kernel
+dnf-utils
+rng-tools
+centos-release
+
+# pull firmware packages out
+-aic94xx-firmware
+-alsa-firmware
+-alsa-lib
+-alsa-tools-firmware
+-ivtv-firmware
+-iwl1000-firmware
+-iwl100-firmware
+-iwl105-firmware
+-iwl135-firmware
+-iwl2000-firmware
+-iwl2030-firmware
+-iwl3160-firmware
+-iwl3945-firmware
+-iwl4965-firmware
+-iwl5000-firmware
+-iwl5150-firmware
+-iwl6000-firmware
+-iwl6000g2a-firmware
+-iwl6000g2b-firmware
+-iwl6050-firmware
+-iwl7260-firmware
+-libertas-sd8686-firmware
+-libertas-sd8787-firmware
+-libertas-usb8388-firmware
+
+# Some things from @core we can do without in a minimal install
+-biosdevname
+-plymouth
+-iprutils
+
+# enable rootfs resize on boot
+cloud-utils-growpart
+gdisk
+
+# add insight-clients
+insights-client
```

Comment:
why do you include that for centos? Isn't that package rhel exclusive (and only makes sense with a rhel subscription)? https://wiki.centos.org/Manuals/ReleaseNotes/CentOS8.1905#head-79555e73a6ccc7e2455a515437302a2084380ba0

Reply:
hmm its in http://olcentgbl.trafficmanager.net/centos/8.0.1905/AppStream/x86_64/os/ :) but still I'd guess without the serverside part it does not make much sense, right?

Reply:
Yeah, I can remove that.  It was copy/paste from a RHEL8 kickstart.  Ideally for consistency I'd like to keep the CentOS images somewhat similar to what is being published for RHEL, although we're not bound by that.

Reply:
I agree, I would absolutely follow the rhel8 kickstart - but insights client only makes sense if you push the insights reports to redhat to get analysed there.